packages/ibm-products/src/components/CreateFullPage/CreateFullPageStep.tsx

```diff
@@ -28,6 +30,90 @@ const defaults = {
   includeStep: true,
 };
 
+interface CreateFullPageStepBaseProps {
+  /**
+   * Content that shows in the CreateFullPage step
+   */
+  children?: ReactNode;
+
+  /**
+   * Sets an optional className to be added to the CreateFullPage step
+   */
+  className?: string;
+
+  /**
+   * Sets an optional description on the progress step component
+   */
+  description?: ReactNode;
+
+  /**
+   * This will conditionally disable the submit button in the multi step CreateFullPage
+   */
+  disableSubmit: boolean;
```

Comment:
This should be optional
```suggestion
  disableSubmit?: boolean;
```